search.nim

```diff
@@ -403,6 +404,9 @@ proc replaceAll*(win: var utils.MainWin, find, replace: cstring): int =
       found = ret[2]
   
     if found:
+      if addr(startMatch).getOffset() <= lastPos:
+        break
+      lastPos = addr(startMatch).getOffset()
```

Comment:
Hrm. I'm wondering if this will break the following scenario:

```nim
var x = 5
var y = 4
# Cursor is here.
var x = 5
var x = 5
```

I replace all ``x`` with ``blah``. Will all x's still be replaced? or just the two below the cursor?